index.js

```diff
@@ -44,7 +32,7 @@ module.exports = function randomAccessKeychain(service, account) {
     },
     write: function(req) {
       getPassword(function(err, existing) {
-        if (err && err.code === 'PasswordNotFound') {
+        if (existing === undefined) {
```

Comment:
The [docs for keytar](https://www.npmjs.com/package/keytar) say that it returns `null` when a password doesn't exist, so it would make more sense to do `existing == null` (note the two `==` - this catches `null` and `undefined`).